src/Login.js

```diff
@@ -24,34 +26,28 @@ function Login() {
     setOpenSnackbar(false);
   };
 
-  const handleSubmit = (event) => {
+  const handleSubmit = async (event) => {
     event.preventDefault();
+    setLoading(true);
 
-    // Obtient les données du formulaire
     const formData = new FormData(event.target);
-    const email = formData.get('email');
+    const username = formData.get('email'); // Utilisation de 'email' pour le champ 'username'
     const password = formData.get('password');
 
-    // Valide le username et le password
-    if (email === 'maurice-tran' && password === 'correct') {
-      // Crée l'objet utilisateur
-      const user = {
-        id: uuidv4(),
-        email: email,
-        password: password,
-        isSuperadmin: true,
-      };
+    try {
+      const response = await axios.post(`${process.env.REACT_APP_BACKEND_URL}/log/in`, { username, password });
 
-      // Store the user object in localStorage
-      localStorage.setItem("users", JSON.stringify(user));
-
-      // Appelle la fonction de login avec l'email
-      login(email);
-
-      // Redirect to the /features page
-      navigate("/features");
-    } else {
+      if (response.status === 200) {
+        localStorage.setItem("session_token", response.data.access);
+        login(username);
+        navigate("/features");
+      } else {
+        setOpenSnackbar(true);
+      }
```

Comment:
Uses early return pattern to reduce the indentation level
```suggestion
      if (response.status === 200) {
        localStorage.setItem("session_token", response.data.access);
        login(username);
        navigate("/features");
        return;
      } 
      
       setOpenSnackbar(true);
```